#! /bin/sh /usr/share/dpatch/dpatch-run
## 61_use_agent_default.dpatch by  <scott@kitterman.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use agent by default. (LP: #15485)

@DPATCH@
diff -urNad gnupg-1.4.9~/g10/options.skel gnupg-1.4.9/g10/options.skel
--- gnupg-1.4.9~/g10/options.skel	2008-07-21 01:03:49.000000000 +0200
+++ gnupg-1.4.9/g10/options.skel	2008-07-21 01:19:59.000000000 +0200
@@ -198,8 +198,12 @@
 # at ftp.gnupg.org/gcrypt/alpha/aegypten/).  To make use of the agent,
 # you have to run an agent as daemon and use the option
 #
-# use-agent
-# 
+# For Ubuntu we now use-agent by default to support more automatic
+# use of GPG and S/MIME encryption by GUI programs.  Depending on the
+# program, users may still have to manually decide to install gnupg-agent.
+
+use-agent
+ 
 # which tries to use the agent but will fallback to the regular mode
 # if there is a problem connecting to the agent.  The normal way to
 # locate the agent is by looking at the environment variable
